src/components/NewPost/NewPost.js

```diff
@@ -20,22 +20,22 @@ const NewPost = ({setNewPostToggle}) => {
         })
         .then(res=>res.json())
```

Comment:
You use this endpoint several time through your code - I think it would worth to encapsulate this call into its own function like `postImageCloudinary` or something like that  and it would return the url - you could use that thourghout your code where you need to upload picture